python/samples/learn_resources/evaluate_with_prompt_flow.py

```diff
@@ -28,8 +28,8 @@
 connection = AzureOpenAIConnection(
     name="AzureOpenAIConnection",
     type="Custom",
-    api_key=config.get("AZURE_OPEN_AI__API_KEY", None),
-    api_base=config.get("AZURE_OPEN_AI__ENDPOINT", None),
+    api_key=config.get("AZURE_OPENAI__API_KEY", None),
```

Comment:
This should be `AZURE_OPENAI_API_KEY` with one underscore, instead of two. Same with endpoint, please.

Reply:
fixed!